src/components/input-style/input-style.ts

```diff
@@ -71,19 +86,26 @@ export class MInputStyle extends ModulVue {
         });
     }
 
-    protected created(): void {
-        setTimeout(() => {
-            this.animReady = true;
-            this.setInputWidth();
-        }, 0);
+    private calcLabelOffset(value: boolean): void {
```

Comment:
J'enlèverais le params de la méthode. Parce que calcLabelOffset(value) c'est pas clair c'est quoi la value attendu quand je lis la signature de la méthode, et aussi parce que value sera toujours this.isLabelUp.